### DISCUSSION TITLE
why `this` has no type in actions

### DISCUSSION THREAD

**Discussion by FuYou00** [vuejs/pinia#1809] [Help and Questions]
why `this` has no type in actions

![image](https://user-images.githubusercontent.com/58352334/202181350-3cad189c-dded-4a42-b527-aca4ae252a29.png)
![image](https://user-images.githubusercontent.com/58352334/202181459-62416892-d47c-457d-ba50-d85ab460bce3.png)

**Comment by mgd216** [comment]
I'm also seeing "this" issue when an action is trying to reference the state:

![image](https://user-images.githubusercontent.com/617733/206193085-607ae199-248a-4517-aa58-4c8427f983d6.png)

I'm using
- Node 18.11
- Vue 3
-  Nuxt 3  (tsconfig.json is default from Nuxt)

**Reply by kongweigen** [reply]
现在解决了么
